exercises/hamming/description.md

```diff
@@ -30,5 +30,5 @@ The Hamming distance between these two DNA strands is 7.
 # Implementation notes
 
 The Hamming distance is only defined for sequences of equal length. This means
-that based on the definition, each language could deal with getting sequences
-of equal length differently.
+that based on the definition, each language could deal with handling sequences
+of different length differently.
```

Comment:
I like the symmetry that @chastell points out here.

Reply:
> (they’re different → so they’re handled differently

To me, that wording would make the most sense if the point of this sentence was to say something similar to "strings of unequal lengths are handled in a different way than strings of equal lengths"

To me, this sentence instead says "each language has its own way of dealing with strings of unequal lengths"

Note that this comment only points out that I find the symmetry to not apply and thus this proposal is no better than any other. However, since it is also no worse than any other, no particular change is requested of this PR. If I don't hear anything by the time 3*24 hours have passed since PR submission, I would merge it as is. There are about 24+7 hours left for that, if I haven't made a mistake.

Anyone feel free to ask for an extension on that time if necessary.

Reply:
I would suggest that "each language could deal with handling sequences of different length in a different way" might be more understandable, but I could live with the current wording unless someone else has a stronger opinion on this.

Reply:
How about spelling what we mean out?

> The Hamming distance is only defined for sequences of equal length, so an attempt to calculate it between sequences of different lengths should not work. The general handling of this situation (e.g., raising an exception vs returning a special value) may differ between languages.

Reply:
This is my favorite so far. 💯

Reply:
Rad, so I think my deal on this PR is: Since it looks like all parties who have commented so far are in agreement with the proposed revision in https://github.com/exercism/problem-specifications/pull/1242/files#r188083656 , if that change is made, we should merge this PR at that point.

If that change is **not** made (no need to speculate on reasons why it wouldn't be made) and someone feels impatient, since the parties also agreed that the current wording is acceptable, it should just be merged by whoever is feeling impatient. Note that I am not currently impatient and do not expect to be for several months, but I won't presume to speak for others.